src/main/java/dev/galacticraft/mod/Galacticraft.java

```diff
@@ -65,50 +66,49 @@
  * @author <a href="https://github.com/TeamGalacticraft">TeamGalacticraft</a>
  */
 public class Galacticraft implements ModInitializer {
+    public static final Registry<VillagerProfession> MOON_VILLAGER_PROFESSION_REGISTRY = new SimpleRegistry<>(RegistryKey.ofRegistry(new Identifier(Constant.MOD_ID, "moon_villager_profession")), Lifecycle.stable());
 
-    public static final Registry<VillagerProfession> MOON_VILLAGER_PROFESSION_REGISTRY = new SimpleRegistry<>(RegistryKey.ofRegistry(new Identifier(Constants.MOD_ID, "moon_villager_profession")), Lifecycle.stable());
-
-    public static final Logger logger = LogManager.getLogger("Galacticraft");
+    public static final Logger LOGGER = LogManager.getLogger("Galacticraft", new GalacticraftPrependingMessageFactory());
 
-    public static final ConfigManager configManager = new ConfigManagerImpl();
+    public static final ConfigManager CONFIG_MANAGER = new ConfigManagerImpl();
 
     @Override
     public void onInitialize() {
         long startInitTime = System.currentTimeMillis();
-        logger.info("[Galacticraft] Starting initialization.");
-        GalacticraftFluids.register();
-        GalacticraftBlocks.register();
-        GalacticraftBlockEntities.init();
+        LOGGER.info("Starting initialization.");
+        GalacticraftFluid.register();
+        GalacticraftBlock.register();
+        GalacticraftBlockEntityType.register();
         GalacticraftItems.register();
-        GalacticraftTags.register();
+        GalacticraftTag.register();
         GalacticraftRecipes.register();
         GalacticraftEntityTypes.register();
-        GalacticraftLootTables.register();
-        GalacticraftStructures.register();
-        GalacticraftFeatures.register();
+        GalacticraftLootTable.register();
+        GalacticraftStructure.register();
+        GalacticraftFeature.register();
         GalacticraftSurfaceBuilders.register();
-        GalacticraftCarvers.register();
+        GalacticraftCarver.register();
         GalacticraftBiomes.register();
-        GalacticraftBiomeSources.register();
+        GalacticraftBiomeSource.register();
         GalacticraftDimensions.register();
         GalacticraftScreenHandlerTypes.register();
-        GalacticraftParticles.register();
-        GalacticraftCommands.register();
-        GalacticraftS2CPacketReceivers.register();
-        GalacticraftSounds.register();
-        GalacticraftBannerPatterns.register();
+        GalacticraftParticle.register();
+        GalacticraftCommand.register();
+        GalacticraftServerPacketReceivers.register();
```

Comment:
Would this be `GalacticraftServerPacketReceiver` to go with the singular naming scheme?

Reply:
Actually now I see a few more plural things:
![image](https://user-images.githubusercontent.com/26600014/115604324-3d847700-a2af-11eb-8106-fc2d31eab99c.png)
![image](https://user-images.githubusercontent.com/26600014/115604345-45441b80-a2af-11eb-86e1-cfec6d69d0eb.png)
![image](https://user-images.githubusercontent.com/26600014/115604377-4b39fc80-a2af-11eb-966d-edadeb61c41f.png)
![image](https://user-images.githubusercontent.com/26600014/115604403-52f9a100-a2af-11eb-907e-bbd0f02046bd.png)
![image](https://user-images.githubusercontent.com/26600014/115604424-5ab94580-a2af-11eb-8aba-9e2fb61d90ae.png)